google/cloud/logging/handlers/container_engine.py

```diff
@@ -21,7 +21,12 @@
 
 import logging.handlers
 
-from google.cloud.logging.handlers._helpers import format_stackdriver_json
+from google.cloud.logging.handlers._helpers import format_stackdriver_json, get_trace_id
```

Comment:
One import per line, please.
```suggestion
from google.cloud.logging.handlers._helpers import format_stackdriver_json
from google.cloud.logging.handlers._helpers import get_trace_id
```